src/js/base/fontawesome.js

```diff
@@ -24,4 +24,5 @@ Handlebars.registerHelper({
   far: _.partial(faHelper, 'far'),
   fas: _.partial(faHelper, 'fas'),
   fal: _.partial(faHelper, 'fal'),
+  fa: faHelper,
```

Comment:
It's moderately unfortunate how simple this was considering some of the `isFas` logic stuff